srcpkgs/ruby-build/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'ruby-build'
 pkgname=ruby-build
-version=20211227
+version=20221123
```

Comment:
another new version, 20221124, is available :|

Reply:
The version has been updated again. ruby-build contains a lot of ruby version definitions, and every time a new definition is added or is changed, a new version is published.